encoding/csv.ts

```diff
@@ -25,7 +25,8 @@ export {
 export type { ReadOptions } from "./csv/_io.ts";
 
 const QUOTE = '"';
-export const NEWLINE = "\r\n";
+const LF = "\n";
+export const CRLF = "\r\n";
```

Comment:
Maybe let's not export this, but define another `CRLF` in testing to avoid this being considered as a public API.